backend/requirements.txt

```diff
@@ -20,3 +20,4 @@ pytest-cov==5.0.0
 pytest-django==4.8.0
 pytz==2024.2
 sqlparse==0.5.1
+django-filter==24.3
```

Comment:
You mention that you removed unnecessary dependencies. Is django-filter still required, or does the hot function work without the requirement?